.github/workflows/pr-check-shell.yaml

```diff
@@ -0,0 +1,15 @@
+name: Check Shell Code
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  codescan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run ShellCheck
+        uses: ludeeus/action-shellcheck@master
+        with:
```

Comment:
we don't have the path `build/ubuntu-22.04`, to modify the yaml to fit CNAP, pleaes refer to https://github.com/marketplace/actions/shellcheck

Reply:
Thanks, fixed.